src/elements/content-uploader/stories/tests/ContentUploader-visual.stories.js

```diff
@@ -0,0 +1,220 @@
+
+        const cancelButtons = canvas.getAllByLabelText('Cancel this upload');
+        await userEvent.click(cancelButtons[1]); // Cancel middle one
+
+        expect(canvas.queryByText('upload_file_1.txt')).toBeInTheDocument();
+        expect(canvas.queryByText('upload_file_2.txt')).not.toBeInTheDocument();
+        expect(canvas.queryByText('upload_file_3.txt')).toBeInTheDocument();
+    },
+};
+
+export const successfulMultipleUpload = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const dropzone = canvas.getByTestId('upload-input');
+        await userEvent.upload(dropzone, multipleFiles);
+
+        const uploadButton = canvas.getByText('Upload');
+        await userEvent.click(uploadButton);
+
+        await waitFor(
+            () => {
+                expect(canvas.getByText('Success! Your files have been uploaded.')).toBeInTheDocument();
+                expect(canvas.getByText('Select More Files')).toBeInTheDocument();
+            },
+            {
+                timeout: SLEEP_TIMEOUT,
+            },
+        );
+    },
+};
+
+export const errorSingleUpload = {
+    play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+
+        const dropzone = canvas.getByTestId('upload-input');
+        await userEvent.upload(dropzone, singleFile);
+
+        const uploadButton = canvas.getByText('Upload');
+        await userEvent.click(uploadButton);
+
+        const cancelButton = canvas.getByLabelText('Cancel this upload');
+        expect(cancelButton).toBeInTheDocument();
+
+        await waitFor(
+            () => {
+                expect(canvas.getByText('Something went wrong with the upload. Please try again.')).toBeInTheDocument();
+            },
+            {
+                timeout: 35000, // gotta wait for the tries to fail before the error message shows
+            },
+        );
+    },
+    parameters: {
+        msw: {
+            handlers: [
+                http.options(`${DEFAULT_HOSTNAME_API}/2.0/files/content`, () => {
+                    return new HttpResponse(null, { status: 401 });
+                }),
+                http.post(`${DEFAULT_HOSTNAME_UPLOAD}/api/2.0/files/content`, () => {
+                    return new HttpResponse(null, { status: 401 });
+                }),
+            ],
+        },
+    },
+};
+
+export default {
+    title: 'Elements/ContentUploader/tests/visual',
+    component: ContentUploader,
+    args: {
+        features: global.FEATURE_FLAGS,
+        rootFolderId: global.FOLDER_ID,
+        token: global.TOKEN,
+    },
+    parameters: {
+        ...defaultVisualConfig.parameters,
+        msw: {
+            handlers: [
+                http.options(`${DEFAULT_HOSTNAME_API}/2.0/files/content`, () => {
+                    return HttpResponse.json({});
+                }),
+                http.post(`${DEFAULT_HOSTNAME_UPLOAD}/api/2.0/files/content`, () => {
```

Comment:
i know that having the /api/ in the path vs subdomain works, but should it? or is there any reason to have the /api/ in there?

Reply:
There is only one location outside of this file that has the work api next to the word, dont know if its worth actually creating a new constant for it

Reply:
i think it's more "proper" from a backend routing perspective to _not_ have the /api/ in the path. would you be ok to remove it, if the tests still pass?

Reply:
/api/ is part of the official route its mocking

Reply:
yup confirmed